### ISSUE TITLE
External Libraires only show jdk lib.

### ISSUE THREAD

**Issue by Jungle666** [dpoluyanov/bazel-wsl#3]
Help~

![image](https://user-images.githubusercontent.com/9640834/146028149-ae4a4cd4-febc-4d57-9fb5-9849dbf54eae.png)

**Comment by Jungle666** [comment]
And The package were not imported success.
![image](https://user-images.githubusercontent.com/9640834/146028751-b3df93ac-90f9-4e6f-ba23-d5e13b10e34a.png)

**Comment by dpoluyanov** [comment]
probably it is the case od https://github.com/bazelbuild/intellij/issues/3112
I’ve created git patch for original repository, and locally use the patches version of intel’s bazel plugin. Probably could rebuild on top of master & share patched plugin at Friday

**Comment by dpoluyanov** [comment]
https://github.com/dpoluyanov/bazel-wsl/releases/download/v0.0.2/ijwb_bazel-fix-3112.zip rebuild it from master

**Comment by Jungle666** [comment]
> https://github.com/dpoluyanov/bazel-wsl/releases/download/v0.0.2/ijwb_bazel-fix-3112.zip rebuild it from master

It's resoved extend lib problems?

**Comment by dpoluyanov** [comment]
Yep, it should resolve an issue with dependencies, but try next version of Bazel-wsl and ijwb https://github.com/dpoluyanov/bazel-wsl/releases/download/v0.0.3/ijwb_bazel-fix-3112.zip with additional fixes for run configurations

**Comment by Jungle666** [comment]
Thanks a lot,after  download，how to use it？

**Comment by dpoluyanov** [comment]
Download & install it into IntelliJ IDEA through: File -> Settings -> Plugins -> ⚙ (Manage repositories) -> Install plugin from disk (alternatively Help -> Find Action, type Install plugin from disk and press Enter), and choose downloaded zip archive as plugin for installation.

**Comment by Jungle666** [comment]
I tried idea 2021.2 and 2021.3.
The version is older or newer.
![image](https://user-images.githubusercontent.com/9640834/146740250-2ca9e222-b13a-4605-9f31-7d531e091b25.png)

![image](https://user-images.githubusercontent.com/9640834/146740076-5f7aa755-c749-42db-afc3-6fec47fe69a0.png)

**Comment by dpoluyanov** [comment]
As for https://plugins.jetbrains.com/plugin/8609-bazel/versions/stable/149663 the minimum compatible range is 212.5080.55 (I've tested on IntelliJ 2021.2.3), but your version is 212.4746.92, so I'd suggest to update IntelliJ to 2021.2.3

**Comment by Jungle666** [comment]
great! Finally success on my wsl!

**Comment by Jungle666** [comment]
> As for https://plugins.jetbrains.com/plugin/8609-bazel/versions/stable/149663 the minimum compatible range is 212.5080.55 (I've tested on IntelliJ 2021.2.3), but your version is 212.4746.92, so I'd suggest to update IntelliJ to 2021.2.3

By the way, is there any way support idea 2021.3 version?

**Comment by dpoluyanov** [comment]
https://github.com/bazelbuild/intellij/issues/3001 an issue still open in bazel-IntelliJ plugin, so I guess work in progress

**Comment by Jungle666** [comment]
OK，wait for a good news.
